ChangeLog.md

```diff
@@ -5,7 +5,7 @@ Emperor 0.9.51-dev (changes since Emperor 0.9.5 go here)
 --------------------------------------------------------
 
 ### Bug Fixes
-
+* Fix problem where the taxonomic vectors weren't scaled properly when the scale coordinates button was toggled
```

Comment:
Thanks, can you add a period at the end of the sentence and reference the issue number?